packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -138,6 +136,25 @@ describe('when route does not contain a project key', () => {
     });
   });
 });
+describe('when user fist visits "/" with no projectKey defined in localStorage', () => {
+  let operations;
+  beforeEach(() => {
+    operations = mocksForMc.createMockOperations();
+    createGraphqlMockServer(xhrMock, {
+      operationsByTarget: { mc: operations },
+    });
+  });
+  it('should not render the NavBar first, then redirect to "/:projectKey" and render the NavBar', async () => {
```

Comment:
FYI: I wanted to test that the navbar is not shown **before** the redirect, but I can't think of a way to do that. I would still like to keep the test description like this, to make it clear what the intention is.